README.md

```diff
@@ -102,6 +113,7 @@ These are modules or frameworks for virtual machines which run within the Cosmos
 - [Comove](https://github.com/co-move/comove) - Libra's Move Virtual Machine
 - [CosmWasm](https://github.com/CosmWasm/cosmwasm) - WASM Virtual Machine & Rust Smart Contracts
 - [Ethermint](https://github.com/chainsafe/ethermint) - Ethereum Virtual Machine
+- [Kadenamint](https://github.com/kadena-io/kadenamint) - Pact smart contract language on Tendermint
```

Comment:
I think this one should go into the tendermint/awesome repo. I don't think this can run within the Cosmos SDk

Reply:
Makes sense. I'll add Kadenamint and Kepler on tendermint/awesome then.